exchanges/multipart-fetch/src/multipartFetchExchange.ts

```diff
@@ -36,7 +36,10 @@ export const multipartFetchExchange: Exchange = ({
       if (files.size) {
         url = makeFetchURL(operation);
         fetchOptions = makeFetchOptions(operation);
-        if (fetchOptions.headers!['content-type'] === 'application/json') {
+        if (
+          fetchOptions.headers!['content-type'] === 'application/json' ||
```

Comment:
We should still keep `application/json` content type header check for compatibility.